src/App.tsx

```diff
@@ -1,61 +1,130 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './components/UserInfo';
+import { Todo } from './components/TodoInfo';
+import { newTodoId } from './helpers';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [isTitleValid, setIsTitleValid] = useState(false);
+  const [isUserIdValid, setIsUserIdValid] = useState(false);
+  const [userId, setUserId] = useState(0);
+  const [actualTodos, setListTodos] = useState(todos);
```

Comment:
In useStare our variables should have the same name for more readability. 

I mean if the first variable is `actualTodos` then the second should be `setActualTodos`